firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -182,7 +183,10 @@ public synchronized UpdateTask updateApp() {
       return updateTask;
     }
 
-    return this.updateAppClient.updateApp(cachedLatestRelease, currentActivity);
+    UpdateTask updateTask =
+        this.updateAppClient.updateApp(cachedLatestRelease, currentActivity, basicConfiguration);
```

Comment:
What are your thoughts about renaming basicConfiguration to something that more clearly indicates that it is a feature flag? My first thought is `showDownloadInNotificationManager`

Reply:
Agreed, changed it and also added the updateApp overload